packages/ui-components/src/components/DateRangePicker/DateRangePicker.tsx

```diff
@@ -55,11 +56,22 @@ const Button = styled(MuiButton)`
   }
 `;
 
-const Label = styled(Button)`
+const Label = styled(Typography)`
   padding-left: 1rem;
   padding-right: 1rem;
   min-width: 8rem;
-  pointer-events: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: ${props => props.theme.palette.text.secondary};
+  font-size: 1rem;
+  background-color: white;
+  border-left-width: 2px;
+  cursor: auto;
+  margin-left: 0 !important;
```

Comment:
Since this is a ui-component it would be nice to not use !important if possible. What is the important over writing? Can we do this with some mui- selectors instead?